react/config.js

```diff
@@ -187,12 +190,39 @@ class StoreConfig extends React.Component {
   }
 
   handleHideAlert = () => {
-    this.setState({ showAlert: false })
+    const {
+      storeconfig: { initialSetup },
+    } = this.props
+
+    if (initialSetup && initialSetup.emailsSetupSuccess) {
+      this.setState({ showAlert: false, alertMessage: '', alertType: '' })
+      this.props.clearEmaisSetup()
+    }
+
+    if (initialSetup && initialSetup.paymentsSetupSuccess) {
+      const origin = window.location.origin
+      const ruleId = initialSetup.ruleId
+      const url = `${origin}/admin/pci-gateway/#/rules/64/rule/${ruleId}`
```

Comment:
Acho que vale a pena explicar esse magic number.

To chutando o nome dele:

```suggestion
      const CUSTOMER_CREDIT_ID = 64
      const url = `${origin}/admin/pci-gateway/#/rules/${CUSTOMER_CREDIT_ID}/rule/${ruleId}`
```